crazyflie/scripts/crazyflie_server.py

```diff
@@ -67,24 +67,31 @@ def __init__(self):
         self._ros_parameters = self._param_to_dict(self._parameters)
 
         self.uris = []
-        self.cf_dict = {}
+        # for logging, assign a all -> all mapping
+        self.cf_dict = {
```

Comment:
This causes issues for the if statement on line 367

```
if self.swarm.fully_connected_crazyflie_cnt == len(self.cf_dict):
            self.get_logger().info("All Crazyflies are fully connected!")
            self._init_parameters()
            self._init_logging()
            self.add_on_set_parameters_callback(self._parameters_callback)
        else:
```

Reply:
Good catch, I think the whole "augmenting" of cf_dict is super scary. Perhaps I can create an additional dict that is just used for logging?

Reply:
the size of fully connected crazyflie is no longer the same as cf_dict. We should probably do a -1

Reply:
Sure, the question is just did I miss any other place:-)? I did look for all places when of cf_dict being used and clearly I did not see that one. I also tested with that backend, but only on the desk - no flight.

Reply:
I pushed a fix for that and another related issue that I found by going through the code again.